invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/DocumentMetadataTabs.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Divider, Tab } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentRelations, DocumentInfo } from './components';
+import { DocumentTableOfContent } from './components/DocumentTableOfContent';
+import { DocumentConference } from './components/DocumentConference';
+
+export default class DocumentMetadataTabs extends Component {
+  constructor(props) {
+    super(props);
+    this.document = props.documentDetails;
+  }
+
+  renderTabPanes = () => {
+    return [
+      {
+        menuItem: 'Details',
+        render: () => (
+          <Tab.Pane attached={false}>
+            <DocumentRelations
+              relations={this.document.metadata.relations}
+              documentType={this.document.metadata.document_type}
+            />
+            <DocumentInfo metadata={this.document.metadata} />
+          </Tab.Pane>
+        ),
+      },
+      {
+        menuItem: 'Content',
+        render: () => (
+          <Tab.Pane attached={false}>
+            <DocumentTableOfContent
+              toc={this.document.metadata.table_of_content}
+              abstract={this.document.metadata.abstract}
+            />
+          </Tab.Pane>
+        ),
+      },
+      {
+        menuItem: 'Publications',
+        render: () => (
+          <Tab.Pane attached={false}>We wait for the schema!</Tab.Pane>
+        ),
+      },
+      {
+        menuItem: 'Conference',
+        render: () => (
+          <Tab.Pane attached={false}>
+            <DocumentConference
+              conference={this.document.metadata.conference_info}
+              documentType={this.document.metadata.document_type}
+            />
+          </Tab.Pane>
+        ),
+      },
+      {
+        menuItem: 'Notes',
```

Comment:
if you are going to apply Chris suggestion to use strings instead of numbers, please re-use the same :)